### PR TITLE
Remove protractor

### PR DESCRIPTION
**Pull request by axelssonHakan**
### Information
* [x] Contains breaking changes
* [ ] Contains new API(s)
* [x] Contains documentation
* [ ] Contains test

### CODE REVIEW

File: docs/e2e.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # System / End 2 End test
 System Testing is a level of the software testing where a complete and integrated software is tested from an enduser perspective.
 
+## protractor in after-work.js 2.0.0
+The protractor module has been removed in after-work.js 2.0.0 and is needed to be included as a devDependencies. The reason is  mainly to keep the size of
+after-work.js down. If you use npm < 3.x the structure of node_modules will require you to also specify mocha as a devDependencies.
```
Comment (peol) [inline docs/e2e.md]: Why would you need `mocha` if < 3?

Also, how does after-work.js report phantomjs/protractor, console output?

Comment (axelssonHakan) [inline docs/e2e.md]: Mocha will be "installed" under node_modules/after-work.js/node_modules/mocha if you use npm 2.x and protractor in /node_modules/protractor and won't find mocha

File: src/config/conf.js [hunk 2]
```diff
@@ -335,6 +335,9 @@ const config = {
     reporter: path.resolve(__dirname, '../../dist/plugins/reporter/index.js'),
     reporterPlugin,
     enableTimeouts: false,
+    reporterOptions: {
+      html: true,
```
Comment (qlikbez) [inline src/config/conf.js]: remove comma

File: src/config/conf.dev.js [hunk 3]
```diff
@@ -284,7 +284,9 @@ const config = {
     ui: 'bdd',
     reporter: 'spec',
     enableTimeouts: false,
-        // ,bail: true,
+    reporterOptions: {
+      html: true,
```
Comment (qlikbez) [inline src/config/conf.dev.js]: remove comma

**Comment by qlikbez** [comment]
One thing, in [https://github.com/qlik-oss/after-work.js/blob/master/src/config/conf.js](https://github.com/qlik-oss/after-work.js/blob/master/src/config/conf.js) we access to "browser" global object and it comes from the protractor package, we will have some code that references something that it is not inside a project dependency.  From another point of view this PR allow us to manage protractor version, and i think it is a good opportunity.

File: src/config/conf.dev.js (hunk 3 above)
Comment (axelssonHakan) [inline src/config/conf.dev.js]: We have changed to AirBnB code style for the OSS repos and this enforces this :-)

Comment (qlikbez) [inline src/config/conf.dev.js]: 8( .. Ok!

File: docs/e2e.md (hunk 1 above)
Comment (axelssonHakan) [inline docs/e2e.md]: @peol Added a check to the runner
https://github.com/qlik-oss/after-work.js/pull/9/files#diff-4efd20b8ffbd324e6a5cf15aec94f5f2

Will try to do the same for phantomjs

**Pull request merged**